box2d.pro

```diff
@@ -11,11 +11,11 @@ contains(QT_CONFIG, reduce_exports): CONFIG += hide_symbols
 
 # Uncomment the line below to compile qml-box2d plugin with Box2D library, installed in OS
 # or pass the variable to qmake in command line:
-# qmake DEFINES+=BOX2D_SYSTEM
```

Comment:
Why not just set this up so that it is a QMake_Extra command on 
linux!android{
     builddeb.command .... .....
}

If documents and examples get get setup a little better they are also going to have to be -dev and -doc -man pages ect.